Gemfile.lock

```diff
@@ -259,7 +259,8 @@ GEM
     mime-types (3.3.1)
       mime-types-data (~> 3.2015)
     mime-types-data (3.2021.0225)
-    mimemagic (0.3.5)
+    mimemagic (0.3.7)
```

Comment:
any particular reason you didn't go 0.3.9?

Reply:
I was just doing what the cool kids were doing:
https://github.com/alphagov/account-api/pull/27/files

Reply:
I think this solves our initial "can't deploy" problem then can update further tomorrow?

Reply:
actually looking at the changelog I think you are right!
Perhaps these came out after the 2ndline push. Will update now.

Reply:
in the unlikely situation that this app actually uses mimemagic you'll need to go to 0.3.9, 0.3.7 has a bug